build.gradle

```diff
@@ -1,5 +1,5 @@
 group 'uk.ac.ebi.subs'
-version '0.1.2-SNAPSHOT'
+version '0.1.3-SNAPSHOT'
```

Comment:
Since there's no change to the resulting JAR, I personally wouldn't bump the JAR version.